<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDB</class>
 <widget class="QDialog" name="ConnectDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="L_nameHost">
       <property name="text">
        <string>Имя хоста</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_nameHost"/>
     </item>
     <item>
      <widget class="QLabel" name="L_nameDB">
       <property name="text">
        <string>Имя базы данных</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_nameDB"/>
     </item>
     <item>
      <widget class="QLabel" name="L_login">
       <property name="text">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_login"/>
     </item>
     <item>
      <widget class="QLabel" name="L_password">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_password">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_portConnectToDB">
       <property name="text">
        <string>Порт подключения к базе данным</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sB_portConnectToDB"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectDB</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectDB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
